### ISSUE TITLE
Şifre Alanları 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ahmet328/sambapos#34]
```
Kullanıcıların şifrelerini yazdıktan sonra kaydet denildiğinden tekrar 
içine girildiğinde gözükmesin.

Yönetime girip basşka kullanıcıların şifrelerini bakıp illegal işlem 
yapabiliyorlar.
```

Original issue reported on code.google.com by `mehmetak...@gmail.com` on 23 Sep 2011 at 2:04

**Comment by GoogleCodeExporter** [comment]
Original comment by `emree...@gmail.com` on 24 Sep 2011 at 2:42
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 2a1bade5b724.
```

Original comment by `emree...@gmail.com` on 28 Sep 2011 at 4:39
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Kontrol edildi
```

Original comment by `mehmetak...@gmail.com` on 4 Oct 2011 at 12:02
- Changed state: **Verified**
